gnovm/pkg/gnolang/machine.go

```diff
@@ -51,6 +51,8 @@ type Machine struct {
 	NumResults int   // number of results returned
 	Cycles     int64 // number of "cpu" cycles
 
+	Debugger
```

Comment:
I suggest moving embedded types at the top of the struct.

Reply:
Instead, I made the field explicit. No real need to embed it.